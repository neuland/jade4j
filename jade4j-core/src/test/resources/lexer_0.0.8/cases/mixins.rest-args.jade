mixin list(tag, ...items)
  #{tag}
    each item in items
      li= item

+list('ul', 1, 2, 3, 4)
