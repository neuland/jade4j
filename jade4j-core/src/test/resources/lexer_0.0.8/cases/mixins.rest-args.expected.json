{"type":"mixin","line":1,"val":"list","args":"tag, ...items"}
{"type":"indent","line":2,"val":2}
{"type":"interpolation","line":2,"val":"tag"}
{"type":"indent","line":3,"val":4}
{"type":"each","line":3,"val":"item","code":"items"}
{"type":"indent","line":4,"val":6}
{"type":"tag","line":4,"val":"li","selfClosing":false}
{"type":"code","line":4,"val":"item","escape":true,"buffer":true}
{"type":"outdent","line":6}
{"type":"outdent","line":6}
{"type":"outdent","line":6}
{"type":"call","line":6,"val":"list","args":"'ul', 1, 2, 3, 4"}
{"type":"newline","line":7}
{"type":"eos","line":7}